src/Websites/Websites.Test/LiveTests/TestLiveScenarios.ps1

```diff
@@ -0,0 +1,150 @@
+Invoke-LiveTestScenario -Name "Create new webapp" -Description "Test creating a new webapp with all default values" -ScenarioScript `
+{
+    param ($rg)
+
+    $rgName = $rg.ResourceGroupName
+    $webName = New-LiveTestResourceName
+    $webLocation = "westus"
+
+    $actual = New-AzWebApp -ResourceGroupName $rgname -Name $webName -Location $webLocation
+    Assert-AreEqual $webName $actual.Name
+    Assert-AreEqual $rgName $actual.ResourceGroup
+    Assert-AreEqual "app" $actual.Kind 
+    
+    Assert-True { $actual.ClientAffinityEnabled } "By default ClientAffinityEnabled should be true"
+    Assert-True { $actual.Enabled } "By default Enabled should be true"
+}
+
+Invoke-LiveTestScenario -Name "Create new web with service plan" -Description "Test creating a new web app with service plan" -ScenarioScript `
+{
+    param ($rg)
+
+    $rgName = $rg.ResourceGroupName
+    $webName = New-LiveTestResourceName
+    $webLocation = "westus"
+    $whpName = "WebServicePlanForPSLiveTest"
```

Comment:
Should it be $whpName = New-LiveTestResourceName ?

Reply:
Thanks for reminding!